src/SDKs/ServiceBus/data-plane/tests/Microsoft.Azure.ServiceBus.Performance/Program.cs

```diff
@@ -0,0 +1,104 @@
+﻿using Microsoft.Azure.ServiceBus;
+using Microsoft.Azure.ServiceBus.Core;
+using System;
+using System.Diagnostics;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.ServiceBus.Performance
+{
+    class Program
+    {
+        private static readonly Stopwatch _stopwatch = Stopwatch.StartNew();
+
+        private static readonly byte[] _payload = new byte[1024];
+
+        private static long _messages;
+
+        static async Task Main(string[] args)
+        {
+            var maxInflight = (args.Length >= 1 ? int.Parse(args[0]) : 1);
+            Log($"Maximum inflight messages: {maxInflight}");
+
+            var messages = (args.Length >= 2 ? long.Parse(args[1]) : 10);
+
+            var connectionString = Environment.GetEnvironmentVariable("SERVICE_BUS_CONNECTION_STRING");
```

Comment:
Should we consider using the existing Service Bus namespace for .NET, represented by `AZ_SERVICE_BUS_CONNECTION`?

Reply:
Ideally, I think we would use the same env vars across all languages.  Currently, the .NET functional tests use `AZ_SERVICE_BUS_CONNECTION`, while the JS functional tests use `SERVICEBUS_CONNECTION_STRING`.  I've (unintentionally) added a third env var `SERVICE_BUS_CONNECTION_STRING` to the JS and .NET perf tests.

If we were going to standardize on one of these, I would choose `SERVICE_BUS_CONNECTION_STRING`:

1. I believe the `AZ_` prefix is unnecessary.
2. The service is named "Service Bus" not "ServiceBus", so the words should be separated by `_`.
3. I believe the suffix `CONNECTION_STRING` is more commonly used than just `CONNECTION`.

So if we want to align these, my proposal would be to change both the .NET and JS functional tests to use `SERVICE_BUS_CONNECTION_STRING`.

Reply:
I've no objection to changing them;  the `AZ_` prefix was added to conform to the pattern used by the configuration client library, which was in place when we did the .NET Service Bus migration.

Reply:
I will create issues to update the other tests to `SERVICE_BUS_CONNECTION_STRING`.